### PR TITLE
Add Docker deploy instructions

### LINKED ISSUE #129: Add docker deploy instructions
Add documentation to demonstrate docker deploy setup for gateway.

### PR DESCRIPTION
**Pull request by nghduc97**
**Title:** 
- Add Docker deploy instructions

### CODE REVIEW

**Comment by VisargD** [comment]
Thanks @nghduc97 for the PR. Will merge this very soon.

File: docs/installation-deployments.md [hunk 1]
```diff
@@ -39,7 +39,20 @@ node build/start-server.js
 ```
 
 # Deploy using Docker
-Docs to be written, please help!
+Run using Docker directly:
+```
+sudo docker run -d -p 8787:8787 portkeyai/gateway
```
Comment (VisargD) [inline docs/installation-deployments.md]: - Can we please remove sudo from the command? 
- And we can also add latest tag to make it more explicit `portkeyai/gateway:latest`
- Can you also link the docker image url as a part of comment: https://hub.docker.com/r/portkeyai/gateway

File: docs/installation-deployments.md [hunk 2]
```diff
@@ -39,7 +39,20 @@ node build/start-server.js
...
+```
+
+# Deploy using Docker Compose
+1. Download compose file from the repo:
+```
+wget "https://raw.githubusercontent.com/Portkey-AI/gateway/main/docker-compose.yaml"
+```
+2. Run:
```
Comment (VisargD) [inline docs/installation-deployments.md:53]: Nitpicking very small change: Can we also mention the default container port that will be exposed? Default is 8787

File: docs/installation-deployments.md (hunk 1 above)
Comment (Willisplayedcloud) [reply docs/installation-deployments.md]: No way

**Review by VisargD — APPROVED** [review]

**Comment by VisargD** [comment]
Looks good. Thanks for the PR..! Closes #129

**Pull request merged**